modules/compliance-profile-types.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/co-concepts/compliance-operator-understanding.adoc
+// * security/compliance_operator/co-scans/compliance-operator-supported-profiles.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="compliance_profile_types_{context}"]
+= Compliance Operator profile types
+
+There are three types of compliance profiles available: Platform, Node, and {op-system-first} profiles. These profile types contain rules that check both OpenShift and operating system settings, such as {op-system} configurations. 
+
+Platform:: Platform scans target your {product-title} cluster, checking once per scan to verify settings. For example, confirming APIServer configurations are using strong encryption cyphers is Platform-level rule.
+
+Node:: Node scans target the nodes of the cluster, checking every Node in a cluster to verify settings. For example, confirming kubeconfig files have the correct permissions to meet a compliance standard is a Node-level rule.

```

Comment:
Since `Node` can mean different things depending on if it's for OpenShift or RHCOS, we could break this into two distinct pieces?

1. Platform profiles for evaluating OpenShift, irrespective of host (e.g., `APIServer`, `Scheduler`, etc.)
2. Node profiles for evaluating OpenShift configuration on each host
3. Node profiles for evaluating RHCOS configuration on each host